### DISCUSSION TITLE
小建议：有线连接每次都有个弹窗需要点确认，能否弄个无障碍自动点击？

### DISCUSSION THREAD

**Discussion by LiamJiang** [eiyooooo/Easycontrol_For_Car#9] [Ideas]
有线连接每次都有个弹窗需要点确认，能否弄个无障碍自动点击？就一个小建议，不加也非常好用了，基本实现了小米汽车su7类似的妙享桌面、一键 Pin 应用功能，非常🐂🍺。

**Comment by eiyooooo** [comment]
有线连接要点确认主要是安卓有USB授权机制，root后应该有方法可以去除

无障碍自动点击的话好像有点浪费无障碍权限了，不打算弄😂

 V1.4.10 加入了adb连接持久化，只需要在应用打开的时候授权一次就可以了

另外有线设备与软件关联的那种授权方式我之前测试过，不同设备差异很大，有的设备可以、有的设备闪退，为了通用性不打算使用
